rails_programming/introduction/installing_rails.md

```diff
@@ -35,6 +35,14 @@ We've previously installed Ruby, and now it's time to install Ruby on Rails. It'
 gem install rails -v 5.2.3
 ~~~
 
+Once the installation finishes, you can check if everything went well by running the following command:
+
+~~~bash
+rails -v
+~~~
+
+This should display the version of Rails installed on your system indicating the installation went smoothly.
+
```

Comment:
Made the changes by removing rbenv rehash and introducing the call to rails -v.

Reply:
Good stuff :)